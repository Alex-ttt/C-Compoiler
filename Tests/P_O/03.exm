if

  1

then

  Statements:
    fo
    (
    )
    a
    ++
else

  Statements:
    fo
    (
    )
    a
    --
