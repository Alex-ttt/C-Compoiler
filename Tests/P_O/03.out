 Symbols Table:
   a int
   b int
   fo  void function fo(
  )
    {
      << block is empty >>
    }
   main  int function main(
  )
    {
      Statements:
      if

        1

      then

        {
          Statements:
          fo
          (
          )
          a
          ++
        }
      else

        {
          Statements:
          fo
          (
          )
          a
          --
        }
    }
